tensorflow/core/common_runtime/mkl_layout_pass.cc

```diff
@@ -1694,6 +1694,24 @@ class MklLayoutRewritePass : public GraphOptimizationPass {
     return do_rewrite;
   }
 
+  static bool Maxpool3DGradRewrite(const Node* n) {
+    CHECK_NOTNULL(n);
+    bool do_rewrite = false;
+    for (const Edge* e : n->in_edges()) {
+      // Rewrite only if there is corresponding Maxpool3D, i.e., workspace is
+      // available
+      if (e->dst()->type_string() == csinfo_.max_pool3d_grad &&
+          e->dst_input() == 1 &&
+          e->src()->type_string() ==
+              mkl_op_registry::GetMklOpName(csinfo_.max_pool3d) &&
+          e->src_output() == 0) {
+        do_rewrite = true;
```

Comment:
Nit: We can just `return true;` here and `return false;` outside the `for` loop. (Don't need `do_rewrite`.)

Reply:
The suggestion is much better. I will do the change.